physionet-django/user/urls.py

```diff
@@ -1,12 +1,12 @@
 from django.urls import path, re_path
 from user import views
+from django.conf import settings
 
 urlpatterns = [
     path('login/', views.login, name='login'),
 
     path('logout/', views.logout, name='logout'),
 
-    path('register/', views.register, name='register'),
     re_path(
         '^activate/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
```

Comment:
I'm pretty sure we want to get rid of that path as well - sso has it's own activate path and this one is simply useless if we disable password login.